addOns/ascanrules/src/main/resources/org/zaproxy/zap/extension/ascanrules/resources/Messages.properties

```diff
@@ -41,11 +41,18 @@ ascanrules.testpersistentxssattack.json.name = Cross Site Scripting Weakness (Pe
 ascanrules.testpersistentxssattack.json.desc = A XSS attack was found in a JSON response, this might leave content consumers vulnerable to attack if they don't appropriately handle the data (response).
 ascanrules.testpersistentxssattack.otherinfo.nothtml = Raised with LOW confidence as the Content-Type is not HTML 
 
-ascanrules.testpersistentxssprime.name=Cross Site Scripting (Persistent) - Prime
-ascanrules.testpersistentxssprime.misc=N/A
 
-ascanrules.testpersistentxssspider.name=Cross Site Scripting (Persistent) - Spider
-ascanrules.testpersistentxssspider.misc=N/A
+ascanrules.testpersistentxss1collectoriginalparamvalues.name=Cross Site Scripting (Persistent) - collect original param values
```

Comment:
These names are quite long. I understand the value in in being specific or clear but there must be something we can do here.

@psiinon @thc202 do you think XSS is popular enough in industry to just use the acronym here? 
Ex: `XSS (Persistent) - Resend Original Values`, etc.

Reply:
@kingthorin what about change them to `Persistence - Resend Original Values`? This is a feature used by stored XSS scanner but may also be used by other scanners (SSTI scanner) so a more generic name would fit well.

Reply:
I'd be good with that, if they're going to be used generically then they should be moved to commonlib?

Reply:
XSS would be ok to me.

Reply:
Agreed, if the plan is to have them used by other scan rules we should move to `commolib`, same for the utils classes.

Reply:
Other scanners just need to interact with the current PersistentXSSUtils. I would rename it to PersistentUtils, move it to commonlib, remove unnecessary things and use it just as a way to store reflections between pages. Should the scanners that find the reflections also be moved to the PesistentUtils?